src/util/fifo.h

```diff
@@ -1,69 +1,116 @@
 #pragma once
 
-#include "pa_ringbuffer.h"
+#include <cstdint>
 
 #include "util/class.h"
 #include "util/math.h"
 
+using ring_buffer_size_t = uint32_t;
```

Comment:
can you share some insight why not just `std::size_t`?

Reply:
Also works for me.  Or uint_fast32_t. I gladly adapt to what you think fits the mixxx codebase best.

Reply:
I think for indexing into `std::vector` it always converts to a `std::size_t` anyways, so if we want to avoid that one sign extension instruction we should use whats most compatible with the underlying container (and avoid exposing it if possible).

Reply:
Sure. I’ll change it to size_t

Reply:
using std::size_t, please check and resolve.